content/en/docs/Getting started/_index.md

```diff
@@ -6,31 +6,63 @@ description: >
   What does your user need to know to try your project?
 ---
 
-{{% pageinfo %}}
-This is a placeholder page that shows you how to use this template site.
-{{% /pageinfo %}}
+## Prerequisites
 
-Information in this section helps your user try your project themselves.
+- [git][git_tool]
+- [go][go_tool] version v1.12+.
+- [docker][docker_tool] version 17.03+.
+  - Alternatively [podman][podman_tool] `v1.2.0+` or [buildah][buildah_tool] `v1.7+`
+- [kubectl][kubectl_tool] version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
 
-* What do your users need to do to start using your project? This could include downloading/installation instructions, including any prerequisites or system requirements.
+## Installation
 
-* Introductory “Hello World” example, if appropriate. More complex tutorials should live in the Tutorials section.
+## Install Released OLM
+For installing release versions of OLM, for example version 0.12.0, you can use the following command:
 
-Consider using the headings below for your getting started page. You can delete any that are not applicable to your project.
+```bash
+export olm_release=0.12.0
+kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl apply -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/olm.yaml
+```
 
-## Prerequisites
+Learn more about available releases [here](https://github.com/operator-framework/operator-lifecycle-manager/releases).
 
-Are there any system requirements for using your project? What languages are supported (if any)? Do users need to already have any software or tools installed?
+## Install From Git Repository Master Branch
 
-## Installation
+You can install OLM from the master branch of the [operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager/) repository with the following: 
 
-Where can your user find your project code? How can they install it (binaries, installable package, build from source)? Are there multiple options/versions they can install and how should they choose the right one for them?
+```bash
+kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+kubectl create -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
+```
+You can also clone the entire git repository and use the [Makefile](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile) for deploying OLM locally on [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for development purposes.
 
-## Setup
```

Comment:
Also, it'll be nice to have the setup, and the try it out section added back in. You could leave them blank for now, and we can add them later. I think it'll be a nice touch for us to have these here.

Reply:
On second thought, we could leave the descriptions under these headers for now too, so that it helps us actively think about what we need to write in there later on.